src/test/java/com/google/api/generator/engine/ast/NewObjectValueTest.java

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.google.api.generator.engine.ast;
+
+import static junit.framework.Assert.assertEquals;
+
+import java.awt.List;
+import java.util.Arrays;
+import java.util.Collections;
+import org.junit.Test;
+
+public class NewObjectValueTest {
+  @Test
+  public void simpleNewObjectValue() {
```

Comment:
Could we please keep the test naming convention consistent? As per the following methods, this should be `newObjectValue_simple`.